tests/test_base.py

```diff
@@ -267,7 +281,7 @@ def test_get_token_plain_different_name(self):
         token = str(uuid4())
         config = {"mytoken": token}
         self.assertIsNone(get_token(config))
-        self.assertEqual(get_token(config, token_key="mytoken"), token)
+        self.assertEqual(get_token(config, token_key="mytoken"), token)  # nosec
```

Comment:
I think this slipped in from your other PR #297 .
```suggestion
        self.assertEqual(get_token(config, token_key="mytoken"), token)
```